Assets/XML/Text/Technologies.xml

```diff
@@ -1684,15 +1785,19 @@
 	<TEXT>
 		<Tag>TXT_KEY_TECH_FILM</Tag>
 		<English>Film</English>
-		<French>Film</French>
+		<French>
+			<Text>Cin&#233;ma</Text>
```

Comment:
"Film" here is meant to reference the medium more than "moving pictures", and also include photo film etc. English has the advantage of covering both with one word but cinéma seems to be more in the direction of moving pictures.

Reply:
Makes sense, I'll change it back